javascript/javascript3/week3/lifeSpan.js

```diff
@@ -0,0 +1,46 @@
+//Series duration of my life
+
+const seriesDurations = [
+    {
+      title: 'Game of thrones',
+      days: 3,
+      hours: 1,
+      minutes:0,  
+    },
+    {
+      title: 'Sopranos',
+      days: 3,
+      hours: 14,
+      minutes: 0,
+    },
+    {
+      title: 'The Wire',
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    }
+  ]
+
+    let lifeSpan = ()=>{ // fat function
+
+        let totalLifehours=80*365*24
+        let total=0;
+    for(let i=0;i<seriesDurations.length;i++)
+    {
+        let lifeHours=seriesDurations[i].days*24 + seriesDurations[i].hours+(seriesDurations[i].minutes/60);
+          let spend=( lifeHours/totalLifehours) *100;
+          total=total+spend;
+          
+
+          let y=seriesDurations[i].title;
+           console.log (y +"took"+"="+ spend +"%"+" "+"of my life");
```

Comment:
This works, but consider using template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals